api/userstore/client.go

```diff
@@ -26,6 +28,17 @@ func New(api restapi.Connector) *UserStore {
 	return &UserStore{api: api}
 }
 
+// GetUser get user details by user ID
+func (store *UserStore) GetUser(id string) (user *LocalUser, err error) {
```

Comment:
Please consider a best practice of Golang naming https://golang.org/doc/effective_go#Getters

Role Store API already have methods to get Users. It gonna be confusing to have both.